src/components/LocationSelector/LocationSelector.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const DEFAULT_LOCATION_CIRCLE_RADIUS = 20;
+
+function LocationSelector(props) {
+    /**
+     * Stores an array of selected location objects
+     *
+     * locationShape = {
+     *  id: PropTypes.string.isRequired,
+     *  title: PropTypes.string.isRequired,
+     *  coordinates: PropTypes.shapeOf({ lng: number, lan: number }).isRequired,
+     *  radius: PropTypes.number.isRequired,
+     * };
+     */
+    const [selectedLocations, setSelectedLocations] = React.useState([]);
```

Comment:
maybe it is too early to note, but will have to have a state to manage the modal's state: isOpen